apps/web/src/components/wizard/DescriptionEditor.tsx

```diff
@@ -24,6 +24,7 @@ export function DescriptionEditor({ form }: Props) {
                 <FormItem className="h-full">
                   <RichMarkdownEditor
                     dismissPreview
+                    allowImageUpload
```

Comment:
@PickleNik this needs a new issue as it's kinda scuffed on this view:

![image](https://github.com/typehero/typehero/assets/996134/b976177b-6bbe-42cd-a751-9c262dac0cce)

Reply:
@Hacksore true, pls make an issue, and for the stepper also

as for solutions: position: relative probably needs to be applied to the left side panel for the Uploadthing

and stepper titles need to be shortened on smaller screens